src/Atmos/Model/tendencies_multiphysics.jl

```diff
@@ -0,0 +1,20 @@
+##### Multi-physics tendencies
+
+export Subsidence
+struct Subsidence{PV, FT} <: SourceTendency{PV}
+    D::FT
+end
+
+# Subsidence includes tendencies in Mass, Energy and TotalMoisture equations:
+Subsidence(D::FT) where {FT} = (
+    Subsidence{Mass, FT}(D),
+    Subsidence{Energy, FT}(D),
+    Subsidence{TotalMoisture, FT}(D),
+)
+
+subsidence_velocity(subsidence::Subsidence{PV, FT}, z::FT) where {PV, FT} =
+    -subsidence.D * z
+
+struct Pressure{PV} <: FluxFirstOrderTendency{PV} end
```

Comment:
`Pressure` is a quantity so I don't think we should call the tendency that as well. How about:
```suggestion
struct PressureGradient{PV} <: FluxFirstOrderTendency{PV} end
```
since it is the tendency contribution of the gradient of the pressure.

Reply:
@simonbyrne I'm realizing that `PressureGradient` may only be appropriate for the momentum equation, whereas the pressure term for the energy equation has no gradients (it's a scalar equation). Should we just leave this as `Pressure`? @ilopezgp maybe you can weigh in? Alternatively, we could leave `PressureGradient{PV<:Momentum}` and just add `Pressure{Energy}`. I have no strong opinions on this, so long the names are appropriate.

Reply:
For now, I've added `Pressure` and `PressureGradient` separately, but we can combine them if it's preferred--it's a simple switch.